src/VisualStudio/CSharp/Impl/Options/AdvancedOptionPageControl.xaml

```diff
@@ -390,6 +390,14 @@
                               Content="{x:Static local:AdvancedOptionPageStrings.Option_Automatically_open_stack_trace_explorer_on_focus}" />
                 </StackPanel>
             </GroupBox>
+
+            <GroupBox x:Uid="DocumentOutlineGroupBox" Header="{x:Static local:AdvancedOptionPageStrings.Document_Outline}">
+                <StackPanel>
+                    <CheckBox x:Uid="EnableDocumentOutline"
+                              x:Name="EnableDocumentOutline"
+                              Content="{x:Static local:AdvancedOptionPageStrings.Option_Enable_document_outline_experimental_requires_restart}" />
+                </StackPanel>
+            </GroupBox>
```

Comment:
new user option to opt into trying the feature out while it's in experiemental mode.